server.js

```diff
@@ -18,9 +18,66 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Go to /thoughts to see the data added to this endpoint.");
 });
 
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140 
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+} )
+
+// GET thoughts
+// This endpoint should return a maximum of 20 thoughts, sorted by createdAt to show the most recent thoughts first.
+
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec();
```

Comment:
Is there a reason to why you've chosen to display all the posts in descending order here in the backend, but reversed in the frontend?